chapters/introduction/introduction.tex

```diff
@@ -4,5 +4,35 @@ \chapter{Introduction}\label{ch:introduction}
 
 \todo{for some reason in the TOC the chapter name appears twice. Is that intentional?}
 \todo{What is the thesis about? What is the content of it? What is the Author's contribution to it?}
+\todo{I used game and thesis and program and application and work interchangablye. We should define these terms and use them accordingly.}
+
+The main goal of this thesis is to create a video game that combines the concepts of procedural generation and non-Euclidean geometry.
+Procedural generation is a method used to create the world by using algorithms instead of manually designing it.
+It allows for creating virtually infinite worlds that are unique in every playthrough.
+It was first used in the 1980s in games such as \textit{Elite} \cite{Elite1984} and \textit{Rogue} \cite{Rogue1980} and has achieved a significant increase in popularity with the release of \textit{Minecraft} \cite{Minecraft} in 2011.
+Procedural generation has a long and rich history in the video game industry.
```

Comment:
I don't like the phrasing in sentences 14 and 15, can we replace it with (or similar):
"Unlike procedural generation, which has a long and rich history in the video game industry, the concept of employing non-Euclidean geometries is a relatively novel idea.
It has been used in only few games, most notably ..."